torch/_inductor/triton_heuristics.py

```diff
@@ -152,6 +147,8 @@ def __init__(
         self.mutated_arg_names = mutated_arg_names
         self.configs = configs
         self.heuristic_type = heuristic_type
+        self.device_type = triton_meta['device_type']
+        self.device_interface = get_interface_for_device(self.device_type)
```

Comment:
Please ensure this PR focuses on `WrapperCodeGen` generalization but do not need to touch non-wrapper-code-gen.